internal/drivers/nomad/driver.go

```diff
@@ -768,3 +789,11 @@ func generateDestroyCommand(vm string) string {
 		fi
 	`, ignitePath, vm, ignitePath, vm, ignitePath, vm, ignitePath, vm)
 }
+
+func getNetworkResources(portLabel string, ports []int) []*api.NetworkResource {
```

Comment:
Add a comment here as to what this function does. 
And also comment about why the port label is of the specific format

Reply:
I'll be changing this. We can have the labels as `<vmid>_gitspaces_1`, `<vmid>_gitspaces_2` etc. We don't need specific port names